alibi/explainers/integrated_gradients.py

```diff
@@ -481,3 +483,42 @@ def build_explanation(self,
         data.update(deltas=deltas)
 
         return Explanation(meta=copy.deepcopy(self.meta), data=data)
+
+
+class IntegratedGradientsText(Explainer):
+
+    def __init__(self,
+                 model: Union[tf.keras.Model, 'keras.Model'],
+                 layer: Union[None, tf.keras.layers.Layer, 'keras.layers.Layer'] = None,
+                 method: str = "gausslegendre",
+                 n_steps: int = 50,
+                 internal_batch_size: Union[None, int] = 100,
+                 text_transformer: TextTransformer = None,
+                 ) -> None:
+        self._explainer = IntegratedGradients(model=model,
+                                              layer=layer,
+                                              method=method,
+                                              n_steps=n_steps,
+                                              internal_batch_size=internal_batch_size)
+        self.text_transformer = text_transformer
+
+    def explain(self,
+                X: List[str],
+                baselines: Union[None, int, float, np.ndarray] = None,
+                target: Union[None, int, list, np.ndarray] = None) -> Explanation:
+        tokens = self.text_transformer.texts_to_tokens(X)
```

Comment:
One slightly confusing aspect of *IntegratedGradientsText* is that it cannot be applied to tokenized text. Why not allow X to be of type `Union[List[str], List[List[int]]]` (latter type for tokens) and make tokenization step optional?